include/ck_tile/ops/gemm/kernel/streamk_gemm_kernel.hpp

```diff
@@ -0,0 +1,251 @@
+// Copyright © Advanced Micro Devices, Inc., or its affiliates.
+// SPDX-License-Identifier:  MIT
+
+#pragma once
+
+#include "ck_tile/ops/gemm/kernel/gemm_kernel.hpp"
+#include "ck_tile/ops/common.hpp"
+#include "ck_tile/host/concat.hpp"
+
+namespace ck_tile {
+
+enum StreamKReductionStrategy
+{
+    /// @brief Workgroups atomically add their results to the C tensor
+    Atomic = 0,
+    /// @brief For a given tile in the C tensor, one workgroup accumulates results of other
+    /// contributing workgroups
+    Reduction
+};
```

Comment:
```suggestion
enum StreamKReductionStrategy : uint32_t
{
    /// @brief Workgroups atomically add their results to the C tensor
    Atomic = 0u,
    /// @brief For a given tile in the C tensor, one workgroup accumulates results of other
    /// contributing workgroups
    Reduction = 1u
};
```
Would recommend explicit datatype for enumeration.

Also for commenting, there seems to be a mix of inline (/// ) and block (/* */) style in most of CK code. Do you know if there is any consensus for CK style? :)

Reply:
Thanks for the feedback! It is addressed below:

> Would recommend explicit datatype for enumeration.

I added an explicit datatype for the enum in https://github.com/ROCm/composable_kernel/pull/2681/commits/cda189b1608aaa5f82cced5f3475fcc5c7d9a30c.

>  Do you know if there is any consensus for CK style? 

Looking at the other kernel files, there seems to be some mixing and matching between the two styles with inline being more prominent. So, I have unified the commenting style to use the inline style in https://github.com/ROCm/composable_kernel/pull/2681/commits/89f77c1675c16f5cd7a556d2daf2ed354a53a8ae.